src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { createTodo } from '../api/todos';
+import { USER_ID } from '../api/todos';
+import { useTodosContext } from '../TodoContext';
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoHeader: React.FC<Props> = ({ todos }) => {
+  const {
+    titleField,
+    handleError,
+    setTodos,
+    setTitleField,
+    setTempTodo,
+    setLoadingItemsIds,
+  } = useTodosContext();
+
+  const [isDisabled, setIsDisabled] = useState(false);
+  const normalizedTitleField = titleField.trim();
+
+  const addTodo = (newTodo: Omit<Todo, 'id'>) => {
```

Comment:
I think it will be better to store functions like `addTodo`, `deleteTodo`, `editTodo`  inside our context